src/service/master.ts

```diff
@@ -75,17 +75,39 @@ const handleSignal = (timeout: number): NodeJS.SignalsListener => (signal) => {
   }, waitTimeToForceKill)
 }
 
+const transformArgSingle = (args: string[], name: string, func: (value: string) => any) => {
+  const argPrefix = `--${name}=`
+  return args.map(arg => {
+    if (!arg.startsWith(argPrefix)) {
+      return arg
+    }
+    const value = arg.slice(argPrefix.length)
+    return `${argPrefix}${func(value)}`
+  })
+}
+
+const transformArg = (args: string[], name: string, func: (value: string) => any) => {
+  const alternate = name.includes('-') ? name.replace(/-/g, '_') : name.replace(/_/g, '-')
+  args = transformArgSingle(args, name, func)
+  args = transformArgSingle(args, alternate, func)
+  return args
+}
+
 export const startMaster = (service: ServiceJSON) => {
   const { workers: numWorkers, timeout = GRACEFULLY_SHUTDOWN_TIMEOUT_S } = service
 
   if (service.deterministicVary) {
     process.env.DETERMINISTIC_VARY = 'true'
   }
 
-  // Setup dubugger
+  const settings: ClusterSettings = {
+    execArgv: transformArg(process.execArgv, 'max-old-space-size', size => Math.round(parseInt(size, 10)/numWorkers)),
+  }
```

Comment:
What about this?

```suggestion
  const settings: ClusterSettings = {
    execArgv: transformArg(process.execArgv, 'max-old-space-size', size => Math.round(parseInt(size, 10)/numWorkers)),
    // Setup debugger
    ...(LINKED ? { inspectPort: INSPECT_DEBUGGER_PORT } : {}),
  }
  
```

If you don't like it, at least consider the new line after the code block 😄